src/layers/transform/CMakeLists.txt

```diff
@@ -96,6 +96,11 @@ endif ()
 # Add the subdirectories
 add_subdirectory(cereal_registration)
 
+if (LBANN_HAS_DISTCONV)
```

Comment:
Did we need `NVSHMEM` protection here, too?

Reply:
Yes, for now. Possibly not true in the future